Dockerfile

```diff
@@ -20,29 +19,18 @@ WORKDIR $RAILS_ROOT
 # (the src likely changed and we don't want to invalidate Docker's cache too early)
 # http://ilikestuffblog.com/2014/01/06/how-to-skip-bundle-install-when-deploying-a-rails-app-to-docker/
 COPY Gemfile Gemfile
-
 COPY Gemfile.lock Gemfile.lock
 
 # Prevent bundler warnings; ensure that the bundler version executed is >= that which created Gemfile.lock
 RUN gem install bundler
 
 # Finish establishing our Ruby enviornment
-RUN bundle install --without development test
+RUN bundle install
 
 # Copy the Rails application into place
 COPY . .
 COPY config/database.sample.yml config/database.yml
 
-EXPOSE $EXPOSE_PORT
-VOLUME [ "/public/uploads" ]
```

Comment:
For the sake readability, I think it would be good to have VOLUME section in dockerfile.